stake-pool/py/pyproject.toml

```
@@ -0,0 +1,67 @@
+[project]
+name = 'stake-pool'
+version = '1.0'
+authors = [
+  { name= 'joncinque'},
+]
+description = 'solana labs stake pool reference implementation'
```

Since this is only python bindings and not the program, how about:
```suggestion
description = 'Solana Program Library Stake Pool Program Client Library'
```